content/smart.md

```diff
@@ -0,0 +1,126 @@
+
+### code Console and Sandbox ###
+
+Cerner's implementation of SMART on FHIR ecosystem allows you to register, update, and delete your SMART app from our [code Console](https://code.cerner.com/developer/smart-on-fhir).  Additionally, you can open and test your app from code Console in our Sandbox environment.  Currently, there is a 15-minute waiting period for new or updated app configurations to take effect.  If you cannot open the application after the 15-minute period, post a question to the [Cerner FHIR Developers Google Group](https://groups.google.com/forum/#!forum/cerner-fhir-developers).
+
+### Launch Context ###
+
+See the [Scopes and Launch Content](http://docs.smarthealthit.org/authorization/scopes-and-launch-context/) page for more information on how you can obtain the context. Generally, your app must be able to retrieve SMART context parameters such as patient id, encounter id (if available), tenant, and a few others. You can inspect the OAuth2 access_token value that is returned to see the actual context.
+
+## Testing ##
+
+### code Console ###
+
+For provider and patient facing apps, you can select any patient from a list of patients in our Sandbox environment to begin testing. After selecting a patient to test, you can open your app from a new web browser in [code Console](https://code.cerner.com/developer/smart-on-fhir). Testing from Code Console through the browser helps you start developing and testing your app quickly.
+
+### PowerChart ###
+
+Once you have your app working in code Console and you are committed to taking your app to production, contact us through the application available at [https://code.cerner.com/ehr-api](https://code.cerner.com/ehr-api), and we can provide access to *PowerChart* to test your app. For provider facing apps, PowerChart is the environment where providers will access apps in a production environment, so it is very important to test the apps in *PowerChart*. Access to PowerChart is a benefit only offered to members of our code program.
+
+### HealtheLife ###
+
+Direct to consumers apps may also be launched from our patient portal, HealtheLife. Once our patient portal is available for developers, developers in our program will have the ability to launch SMART applications from HealtheLife. This is how you can test as a consumer accessing your application.
+
+#### SSL Certificate ####
+
+Your app must be reachable through an https endpoint. Also, you need a valid SSL certificate for your site to test your app in *PowerChart*.
+
+#### Public Access ####
+
+In order for *PowerChart* to open and display your app, the URL of your app needs to be publicly accessible. In other words, the URL cannot point to hosts on your internal network.
```

Comment:
This is specifically true for our PowerChart sandbox, and would not be true for an app being developed in a client domain, correct?